tasks.py

```diff
@@ -212,11 +212,11 @@ def release(c):  # pylint: disable=unused-argument
         if pattern.fullmatch(tag.name):
             latest_release = tag.name
             break
-    else:
-        latest_release = None
+        else:
+            latest_release = None
 
-    if latest_release and date_info == latest_release[:7]:
-        increment = str(int(latest_release[8:]) + 1).zfill(2)
+    if date_info == latest_release[1:8]:
```

Comment:
This will raise an exception if latest_release is None?

Reply:
Fixed.